performance_tests/src/pylintrc

```diff
@@ -0,0 +1,45 @@
+[MESSAGE CONTROL]
```

Comment:
I think you can leave this pylintrc and delete the others; it should apply to all files recursively